include/mxnet/ndarray.h

```diff
@@ -82,7 +82,8 @@ class MKLDNNMemory;
 class NDArray {
  public:
   /*! \brief default constructor */
-  NDArray() {
+  NDArray()
+    : entry_(nullptr) {
```

Comment:
doesn't the NodeEntry constructor require 3 arguments?

Reply:
I refactored it.